apps/pod-issuer-server/.husky/pre-commit

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env sh
 cd apps/pod-issuer-server
-yarn build && git add dist
+yarn build
+mkdir dist/public
+cp -r ../../node_modules/@pcd/proto-pod-gpc-artifacts/ dist/public/artifacts/
+git add dist
```

Comment:
Oh, this is putting the artifacts directly into your git repo?  Is that required in order to get them onto the server?
I wouldn't recommend that since it's going to make your repo huge over time.  Big binaries can't be purged from git history without a lot of effort.
Does Vercel have a way to deploy binaries which aren't in the repo?  For Zupass on Render we copy them into a directory which exists in the local repo during deployment, but doesn't get pushed to git: https://github.com/proofcarryingdata/zupass/blob/4011501c7e9ca229f922de85da110cb5defb9274/apps/passport-client/build.ts#L226